sdk/test-utils/recorder-v2/src/index.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as url from "url";
+import { DefaultHttpClient, WebResource, WebResourceLike } from "@azure/core-http";
+
+export class RecordingHttpClient extends DefaultHttpClient {
+  private static s_recordingServerUri = "http://localhost:5000";
```

Comment:
Any reason these need to be static to the class and couldn't just be static `const` variables in the module scope?

Reply:
Could be const.